backend/src/libs/enums/user-table-columns.ts

```diff
@@ -0,0 +1,7 @@
+const UserTableColumns = {
```

Comment:
It's not a correct destination for that enum

Reply:
And please contact with guys which are have the tasks related to user table columns, maybe they are done with it